Angular2-Starter-Project.md

```diff
@@ -0,0 +1,229 @@
+# Angular 2 Starter Project
+
+This tutorial will build an extremely simple seed project with Angular2. We will begin with the most basic application possible and then add on a bit more functionality.
+
+### Overview 
```

Comment:
Always use heading 2 for major section *for the love :heartpulse: of @camperbot* 

```md
## Overview 
```